AnKi/Shaders/Fsr.glsl

```diff
@@ -14,7 +14,7 @@ layout(set = 0, binding = 1) uniform ANKI_RP texture2D u_tex;
 layout(set = 0, binding = 2) writeonly uniform ANKI_RP image2D u_outImg;
 layout(local_size_x = 8, local_size_y = 8) in;
 #else
-layout(location = 0) out Vec3 out_color;
+layout(location = 0) out mediump Vec3 out_color;
```

Comment:
Can you use ANKI_RP instead? ANKI_RP is a macro that may be highp if the compiler forces full precision.